src/components/POLine/Cost/CostForm.js

```diff
@@ -61,8 +61,8 @@ const CostForm = ({
   order,
   initialValues,
   change,
-  required = true,
-  hiddenFields = {},
+  required,
```

Comment:
any reasons to make not required changes?

Reply:
I have just moved it to default props below https://github.com/folio-org/ui-orders/pull/1551/files/2730fb43a84a33e693d7cdd07cf0d6b94923eb3c#diff-baf38169fc987a803638e5b81f7db7388631b04aa64859f2dd877b5850ce43f3R341

Reply:
I understand, but what is the reason?

Reply:
I didn't quite understand, what are you implying?

Reply:
don't do changes if you don't need to change

Reply:
I haven't changed. I have just refactored the code according to our codebase pattern.